### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by alexmgrove**

### CODE REVIEW

**Review by whanley — COMMENTED** [review]
This is good, but can you add `status="verified"` tags to the divs you corrected? Aim for about 25.

File: 1906-12-01.xml [hunk 1]
```diff
@@ -1464,149 +1467,168 @@
...
-                            luxe run- fling between Cairo and I.nxor and vice vers* :</p>
-                <p>Be ween ÇaifO end Beni Souel l'.T. 35;</p>
-                <p>Mit ia P.T. 5» ; Aasiont, P. T. 70 ; Sohar,</p>
-                <p>P.T. 90: N-r Hamadi. P.T., 90; K-teb,</p>
-                <p>P.T. l&lt; o , L x ir. P.T. 100. </p> </div>
+                <div type="section" feature="local" status="verified">
+                    <head> LOCAL AND GENERAL. </head>
+                    <div type="item" status="verified">
+                        <head>Plague of Locusts</head>
+                        <!-- The header was misspelled -->
```
Comment (whanley) [inline 1906-12-01.xml:1474]: if this problem has been solved, you can delete the comment

File: 1906-12-01.xml [hunk 2]
```diff
@@ -1464,149 +1467,168 @@
...
+                            tribes scattered over the Arabian desert East of the Nile.</p>
+                    </div>
+                    <div type="item" status="verified">
+                        <head>Sale of Hotel</head>
+                        <p>M. Allvund stated to have sold the Hotel de la Poste for 210,000 francs
+                            to M.S. Romeo, of Port Said. who has taken it over.</p>
+                    </div>
+                    <div type="item" status="verified">
+                        <head>Jewish Benevolent Society</head>
+                        <!-- Missing on this page before 3.1.6 A nile Wreck>   -->
```
Comment (whanley) [inline 1906-12-01.xml:1496]: has this been solved?

File: 1906-12-01.xml [hunk 3]
```diff
@@ -1464,149 +1467,168 @@
...
+                    </div>
+                    <div type="item" status="verified">
+                        <head>Athletics in the Sudan</head>
+                        <p>A new athletic club has been started at Khartonm. Tbe inauguration was
+                            attended by the Sirdar.</p>
+                    </div>
+                    <div type="item" status="verified">
+                        <head>Bedouin Coneus</head>
+                        <!-- missing information and very bad grammar -->
+                        <p>The census of the Bedonins is the first to be taken in the case of the
```
Comment (whanley) [inline 1906-12-01.xml:1486]: Bedouins

File: 1906-12-01.xml [hunk 4]
```diff
@@ -1464,149 +1467,168 @@
...
+                            were the victors : 1. Carl Cylis beat Sanderfly (11 minutes). 2.
+                            Salasma-Haasib beat Noel le Gaulois (3 minutes) 3. Eigmann beat Sturm (4
+                            minutes). 4.Osttinger beat Finisch (2 minutes). 5. Limousin and
+                            Schackman drawn. to be continued another day. This evening the following
+                            matches have been arranged: Milo v. Clement, Ahmed Aly v. Noel. Mohamed
+                            v. Cvlis, Eigmann v. Finish, Pons v. Vander Berg. Tomorrow there will be
+                            afternoon and evening contests, with 1,000 in prizes.</p>
+                    </div>
+                    <div type="item" status="verified">
+                        <head>Honeaty Rowarded</head>
```
Comment (whanley) [inline 1906-12-01.xml:1569]: there are misspellings in the headline. Don't say it's verified if it isn't!

File: 1906-12-01.xml [hunk 5]
```diff
@@ -1464,149 +1467,168 @@
...
                     <p>Built In 1904. Modern House. Splendid situation. Eectric Light<lb/> Lift,
                         Pension P.T. 50. Arrangements for families. </p>
                     <p>Rooms and Breakfast P.T. 25.— Meals a la Cart </p>
                     <p><measure type="indexNo">26789-26-11-09</measure></p>
                 </div>
-                <div type="item">
+                <div type="item" status="verified">
                     <head>THE IRON MARKET.</head>
                     <p>AMERICA EXCITED, GLASGOW BOOM I SO.</p>
                     <p>Gazotto's Special Sorvloe.)</p>
```
Comment (whanley) [inline 1906-12-01.xml:1600]: spelling mistakes--don't say it's verified if it isn't.

File: 1906-12-01.xml [hunk 6]
```diff
@@ -1793,7 +1816,8 @@
                         late«t thing</p>
                 </div>
                 <div type="item" feature="steamerMovements">
-                    <head>STEAMER MOVEMENTS</head> <!-- mispelled words -->
+                    <head>STEAMER MOVEMENTS</head>
+                    <!-- mispelled words -->
```
Comment (whanley) [inline 1906-12-01.xml:1820]: has this been solved?

File: 1904-12-22.xml [hunk 7]
```diff
@@ -0,0 +1,1703 @@
...
+        <fileDesc>
+            <titleStmt>
+                <title level="m" type="main">Digital Egyptian Gazette</title>
+                <title level="m" type="sub">An encoded transcription</title>
+                <editor>Alex Grove</editor>
+                <principal>Will Hanley</principal>
+            </titleStmt>
+            <editionStmt>
+                <edition>
+                    <date when="2022-09-09">September 9, 2019</date>
```
Comment (whanley) [inline 1904-12-22.xml:15]: also change the date in black

File: 1904-12-22.xml [hunk 8]
```diff
@@ -0,0 +1,1703 @@
...
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+                <idno type="URI">https://github.com/dig-eg-gaz/content/1904-12-22/tei</idno>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1904-12-22">Thursday, July 13, 1905</date>
```
Comment (whanley) [inline 1904-12-22.xml:27]: also change the date in black

File: 1904-12-22.xml [hunk 9]
```diff
@@ -0,0 +1,1703 @@
...
+                    <p>A great economiser and saver of time, health, and strength.</p>
+                    <p>Laundry worries went out of woman's life when Sunlight Soap came into it.</p>
+                    <p>It is the purest Soap going.</p>
+                    <p>Soap</p>
+                </div>
+                <!--Missing 2 Egyptian Gazette ads-->
+                <div type="section">
+                    <head>PORT OF ALEXANDRIA</head>
+                    <p>
+                        <hi rend="bold">The interesting article on the port of Alex andria from the
```
Comment (whanley) [inline 1904-12-22.xml:1027]: remove the bold highlight

File: 1904-12-22.xml [hunk 10]
```diff
@@ -0,0 +1,1703 @@
...
+                    </p>
+                    <p>
+                        <hi rend="bold">Maritime opinion, we find, entirely differs from that of Mr.
+                            Cornish as regards the value of the inside breakwater which he proposes,
+                            as a protection to pontoons utilized after the Marseilles fashion
+                            alongside of ships moored "end on" to the quays, under existing</hi>
+                    </p>
+                    <p>
+                        <hi rend="bold">conditions. The drift of water between the I breakwater and
+                            the quays would be far too</hi>
```
Comment (whanley) [inline 1904-12-22.xml:1074]: surely the page doesn't end like this?

**Review by whanley — CHANGES_REQUESTED** [review]
Good start on the deliverable, Alex. Read over page 3 carefully and make sure it's divided into the appropriate items and they come in the proper sequence. To do this, you need to have your page image and your xml text and make sure they correspond to each other.
As far as verification, there are also some details to be cleared up. If you've dealt with a comment, remove it. Don't change the status of every item in a section like Local and General. Verify the whole section once it's correct.

**Comment by whanley** [comment]
Okay, this is fine. Now how about the deliverables?

**Comment by alexmgrove** [comment]
I'm on it, will be done with a few by the end of the week.
Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by alexmgrove** [comment]
What does this mean?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**